client/src/featureform/get_local.py

```diff
@@ -0,0 +1,400 @@
+        "trainingsets": returned_training_sets_list
+    }
+
+    format_rows([("NAME: ", returned_source["name"]),
+    ("VARIANT: ", returned_source["variant"]), 
+    ("OWNER:", returned_source["owner"]),
+    ("DESCRIPTION:", returned_source["description"]),
+    ("PROVIDER:", returned_source["provider"]),
+    ("STATUS: ", returned_source["status"])])
+    format_new_para("DEFINITION:")
+    print(returned_source["definition"])
+    print("FEATURES:")
+    format_rows("NAME", "VARIANT")
+    for f in returned_source["features"]:
+        format_rows(f["name"], f["variant"])
+    format_new_para("LABELS:")
+    format_rows("NAME", "VARIANT")
+    for l in returned_source["labels"]:
+        format_rows(l["name"], l["variant"])
+    format_new_para("TRAINING SETS:")
+    format_rows("NAME", "VARIANT")
+    for t in returned_source["trainingsets"]:
+        format_rows(t["name"], t["variant"])
+    format_new_para("")
+    return returned_source
+
+def get_training_set_variant_info_local(name, variant):
+    db = SQLiteMetadata()
+    training_sets_list = db.getNameVariant("training_set_variant", "name", name, "variant", variant)
+    if training_sets_list == []:
+        print("Training set not found.")
+        return
+    training_set = training_sets_list[0]
+
+    returned_features_list = get_related_resources("training_set_features", 
+                                                    name_col="feature_name", 
+                                                    var_col="feature_variant", 
+                                                    column1="training_set_name", 
+                                                    name=name, 
+                                                    column2="training_set_variant", 
+                                                    variant=variant)
+
+    returned_training_set = {
+        "name": training_set["name"],
+        "variant": training_set["variant"],
+        "owner": training_set["owner"],
+        "description": training_set["description"],
+        "status": training_set["status"],
+        "label": {
+            "name": training_set["label_name"],
+            "variant": training_set["label_variant"]
+        },
+        "features": returned_features_list
+    }
+    format_rows([("NAME: ", returned_training_set["name"]),
+    ("VARIANT: ", returned_training_set["variant"]),
+    ("OWNER:", returned_training_set["owner"]),
+    ("DESCRIPTION:", returned_training_set["description"]),
+    ("STATUS: ", returned_training_set["status"])])
+    format_new_para("LABEL: ")
+    format_rows([("NAME", "VARIANT"), (returned_training_set["label"]["name"], returned_training_set["label"]["variant"])])
+    format_new_para("FEATURES:")
+    format_rows("NAME", "VARIANT")
+    for f in returned_training_set["features"]:
+        format_rows(f["name"], f["variant"])
+    format_new_para("")
+    return returned_training_set
+
+def get_provider_info_local(name):
```

Comment:
is there a get_provider_info for hosted? Could this just be get_provider_info, or get_local_provider?

Reply:
there is one for hosted